### PR TITLE
Added swirl options

### PR DESCRIPTION
**Pull request by seankross**
Still working on some changes.

### CODE REVIEW

**Comment by seankross** [comment]
As you can see, I removed the `coursesDir` argument from the `install_course` suite of functions. I added the ability to set global options for Swirl by incorporating [`rappdirs`](https://github.com/hadley/rappdirs), which was going to be added to Swirl as a dependency in the future anyway. I'll document the new functions shortly.

File: R/zzz.R [hunk 1]
```diff
@@ -1,3 +1,8 @@
+.onLoad <- function(libname, pkgname){
+  set_swirl_options(courses_dir = file.path(system.file("Courses", package = "swirl")))
```
Comment (seankross) [inline R/zzz.R]: This needs a check to see if the options file already exists.

**Comment by seankross** [comment]
Hi @filipsch, 
Global options can now be set for swirl, including the location of the courses directory. Let me know if these changes are acceptable to you. I'm ready for these changes to be merged.

**Comment by filipsch** [comment]
@seankross 
Hi Sean, 
I had a look at your code; these changes make total sense and are a way cleaner way of handling with different install locations for the swirl courses. I'll merge these changes to the Data-Camp/swirl repo.
Do I have to redo my pull request for swirldev/swirl then?

**Comment by seankross** [comment]
Hi @filipsch,
You do not need to redo your pull request.

**Pull request merged**